### DISCUSSION TITLE
Why is 8.0.8x branch still maintained?

### DISCUSSION THREAD

**Discussion by rogerbriggen-securiton** [dotnet/maui#25725] [General]
Does anybody know what is wrong with SR9 so that 8.0.8x is still maintained even so there is 8.0.9x out quite a while?

**Comment by OvrBtn** [comment]
Not 100% sure but I think there was a accidental regression in SR9 https://github.com/dotnet/maui/pull/25613 and maybe that's the cause.
But I'm not part of MAUI team so I might be wrong.
